modules/va_facilities/README.v1.yml

```diff
@@ -434,7 +414,7 @@ paths:
     get:
       tags:
         - facilities
-      summary: Query facilities based on drive time from a specified location
+      summary: Query which VA health (VHA) facilities are nearby, given a specified location and desired drive time
       description: |
         Retrieve all facilities contained within an isochrone (a polygon composed of the area
```

Comment:
This still seems to imply that facilities are being retrieved based on a user-centered isochrone. We are now retrieving all facilities where the passed in location intersects within a facility's isochrone of duration less than the passed in drive time—which is a mouthful. Would it be better to say something more general like: "Retrieve all facilities reachable by driving within the specified time period."

Reply:
For this, I was more going for what the user could expect to get out of this endpoint vs. how it functions on the inside. Although the polygons are now fixed to the facility itself instead of created on-the-fly based on the provided location, I'm not sure it matters to a user either way?

Reply:
If the user doesn't care about the underlying implementation, then I stick with the suggestion to just explain the final product: "Retrieve all facilities reachable by driving within the specified time period."